plenum/server/node.py

```diff
@@ -3504,7 +3504,8 @@ def reportSuspiciousNode(self,
                                      Suspicions.PPR_STATE_WRONG,
                                      Suspicions.PPR_PLUGIN_EXCEPTION,
                                      Suspicions.PPR_SUB_SEQ_NO_WRONG,
-                                     Suspicions.PPR_NOT_FINAL)):
+                                     Suspicions.PPR_NOT_FINAL,
```

Comment:
Oh, it looks like there is a number of other suspicions missing here....
Can you please add the following to the list:
PPR_AUDIT_TXN_ROOT_HASH_WRONG
PPR_BLS_MULTISIG_WRONG
PPR_TIME_WRONG
DUPLICATE_PPR_SENT

Reply:
sure, fixed